### DISCUSSION TITLE
Just downloaded Artisan, but it won't launch

### DISCUSSION THREAD

**Discussion by natellerandi** [artisan-roaster-scope/artisan#941] [General]
Hello, I just downloaded Artisan and it will not run on my PC.  It says this PC, which I just bought, will not support it.  It's an HP Envy x2 tablet with 128gb storage and 4gb RAM, running Windows 10 Home.

Any ideas why I was able to download and set up Artisan but cannot run it?

Thanks!
Nate

**Comment by roasterdave** [comment]
What is the exact error message and when do you see it?  There is a message that might present itself during install if the build is for a different version of Windows.  However, you also say you were able to set up Artisan so I am a bit confused.

Will you confirm the file you downloaded was `artisan-win-2.6.0.zip`

**Comment by natellerandi** [comment]
Hi Dave,

Sorry, I downloaded and installed. Not set up yet. My bad. 

When I then launch it to try to use it, I get a blue pop-up that says: “This app cannot run on your PC. To find a version for your PC, check with the software publisher.”

I verified I downloaded the Windows version. I also tried to download the legacy Windows version but that didn’t work since I’m not running v7 or 8 of Windows. 

Would the fact this is Windows 10 Home have any bearing? 

Thank you,

Nate

**Comment by roasterdave** [comment]
Windows 10 Home is fine for Artisan.  I use it all the time as do a large number of Artisan users.  There is something in the way your system is configured that is preventing Artisan from running.  

Will Artisan run in Administrator Mode?  Go to \Program Files\Artisan and right click on artisan.exe.  Choose 'Run as administrator'.  Click 'Yes' if UAC pops up asking if you want allow this program.

If it does not run, please report the following.  All can be found in the system About box (one way to open it is to type 'about' in the Windows search box and choose 'About your PC').  Copy and paste the entire strings found for 'Device Specifications System Type', 'Windows Specifications Edition' and 'Windows Specifications Version'.  From these we can tell if there is a system incompatibility.  Otherwise it will be some Windows setting that is the problem.

**Reply by natellerandi** [reply]
Hi Dave, thank you for this.  Still, no luck - same error message when I try to run artisan.exe.

Here is the information you requested:
Device Specifications
Device name	DESKTOP-FCT9TT4
Processor	Snapdragon(TM) 835   2.21 GHz
Installed RAM	4.00 GB (3.69 GB usable)
Device ID	1C6177D5-6585-447D-AA75-996AE6D787EA
Product ID	00326-03572-34734-AAOEM
System type	64-bit operating system, ARM-based processor
Pen and touch	Pen and touch support with 10 touch points

Windows Specifications:
Edition	Windows 10 Home
Version	20H2
Installed on	‎7/‎21/‎2021
OS build	19042.928
Experience	Windows Feature Experience Pack 120.2212.551.0

Please let me know if you need any additional information in order to help troubleshoot.  I appreciate your efforts here!

Nate
![Screenshot (1)](https://user-images.githubusercontent.com/111610438/185767242-6df8a765-830d-4c65-81c9-bc53c8c6f902.png)

**Reply by natellerandi** [reply]
Hi Dave,

One last thing, in case it matters.  MSFT Defender (endpoint protection) appears to be active.  Could that be part of the issue - maybe it's seeing Artisan as a potential threat?

**Comment by roasterdave** [comment]
I was afraid this might be the problem when I saw you had an Envy x2.  The Qualcomm Snapdragon processor used in your PC will not run the Artisan executable.  Artisan requires an x64 architecture processor as built by Intel and AMD.  The ARM-based architecture used by the Snapdragon is quite different.  Thus a new build of Artisan is needed, specific to the Snapdragon ARM processor that includes a bootloader for the Snapdragon ARM.  Expect there to be other obstacles getting it to run with the various IO drivers and libraries supplied by third parties. 

This is not on the work list for the near term.  It has not come up before.  None of the Artisan team has an ARM based PC to develop and test such version.  Only x64 can be supported with Windows at this time.

**Comment by natellerandi** [comment]
Thanks, Dave!  I’ll return this and get one that supports x64. Appreciate your time and guidance!

Thank you,

Nate

**Comment by roasterdave** [comment]
For multiple reasons I think you will be happier in the long run with an x64, and not just with Artisan.  Cheers.
